src/Http/Http/test/CookieOptionsTests.cs

```diff
@@ -40,6 +40,7 @@ public void CopyCtor_AllPropertiesCopied()
                 case "HttpOnly":
                 case "IsEssential":
                 case "MaxAge":
+                case "Partitioned":
```

Comment:
I think the idea here is to set these to non-default values so we are sure the properties are really copied.